vignettes/how-to/manage-secrets-and-environments.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "How to: Manage Secrets and Environments"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to: Manage Secrets and Environments}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+Secrets and environments are two distinct yet similar things.
+Typically, secrets are credentials for an external service such as a database.
+Environments, on the other hand, are used to manage multiple configurations that we can be easily switched between.
+Rhino recommends a way for working with either one.
+
+
+## Secrets
+Secrets are a confidential information and should not be tracked in your version control system.
+Therefore, a natural place for them are system environment variables.
+Variables set in system environment can be retrieved within your code with `Sys.getenv()`.
+
+R provides a way to easily set environment variables.
+Upon a session start (or restart) R reads `.Renviron` file contents and sets environment variables.
+
+**.Renviron**
+```sh
+# A comment in .Renviron file
+DATABASE_PASSWORD="foobar123!"
+API_KEY="75170fc230cd88f32e475ff4087f81d9"
+```
+
+**global.R**
```

Comment:
This might suggest that you can use a `global.R` file in a Rhino app as you would do in a typical Shiny application. Perhaps just introduce the snippet with something like "The password can be read and used the following way".